tasks/updateconstants.js

```diff
@@ -224,3 +228,11 @@ function expandItemGroup(key, items) {
     return base;
   }
 }
+
+function replaceRepeatedNumbersWithSlashes(strToReplace) {
+  return strToReplace.replace(/(\d{1,3})\/(\1\/){1,2}\1/, '$1');
```

Comment:
Since repeated numbers for each ability level isn't technically wrong perhaps we should just keep it in order to minimize the risk of something breaking the regex.